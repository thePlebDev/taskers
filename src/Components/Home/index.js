import React from 'react';
import styled from 'styled-components'
import {Link} from "react-router-dom";

import LandingImage from '../../Assets/Images/LandingImage.png'
import Info from '../Info'
import Footer from '../Footer'


const Container = styled.div`
    height:100vh;
    width:90vw;
    margin:0 auto;
    font-size:30px;
    display:grid;
    grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
 
    text-align:center;


`

const Title = styled.h1`
    margin-top:5rem;
    margin-bottom:0.5rem;
    color: #73C2FB;
   
`
const Desc = styled.p`
    margin-top:60px;
    font-size:3rem;
    font-weight:400;
`
const Button = styled.button`
    padding:1rem 3rem;
    font-size:1rem;
    color:#73C2FB;
    border:2px solid #73C2FB;
    border-radius:4px;
    outline:none;
    cursor:pointer;
    background:transparent;
`
const SecondContainer = styled.div`
z-index:9999999;

`
const Image = styled.img`
    
    width:120%;
    position:absolute;
    right:0;
    bottom:40%;
    @media only screen and (min-width: 1000px) {
        bottom:0;
    }

    
`


// the purple color #6C63FF

const Desc2 = styled(Desc)`
    font-size:2rem;
    margin-top:40px;
`
const ThirdContainer = styled.div`
    position:relative;
    display:none;
    @media only screen and (min-width: 500px) {
        display:block;
    }
`

const Span = styled.span`
    font-size:12px;
    margin-left:30px;
`





const Home = ()=>{
    return(
        <>
        <Container>
            <SecondContainer>
                <Title>Taskers</Title>
                    <Desc>Time management for busy people</Desc>
                    <Desc2>Easy to use time Management software for people on the go</Desc2>  
                    <Link to="/main">
                         <Button>Try Taskter today</Button>
                    </Link>              
                
            </SecondContainer>
           <ThirdContainer>
               <Image src={LandingImage}/>
           </ThirdContainer>
        </Container>
        <Info/>
        <Footer/>
        </>
    )

}


export default Home