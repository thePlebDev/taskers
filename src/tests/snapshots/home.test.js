import React from 'react' 
import { BrowserRouter as Router } from "react-router-dom"
import { render } from '@testing-library/react' // the custom render wasnt working 
import GlobalStyle from '../../Styles/global'

import Home from '../../Components/Home'

it("should render the snapshot",()=>{
    const { asFragment } = render(<Router>
                                    <GlobalStyle/>
                                    <Home/>
                                 </Router>)

     const obj = render(<Home/>)
    
   expect(asFragment()).toMatchSnapshot()
})