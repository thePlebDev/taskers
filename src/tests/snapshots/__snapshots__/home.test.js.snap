// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-crHlIS cfoyXr"
  >
    <div
      class="sc-hBURRC holJXx"
    >
      <h1
        class="sc-egiSv kXUkOm"
      >
        Taskers
      </h1>
      <p
        class="sc-bqiQRQ esZgiB"
      >
        Time management for busy people
      </p>
      <p
        class="sc-bqiQRQ sc-fFehDp esZgiB bCBDUG"
      >
        Easy to use time Management software for people on the go
      </p>
      <a
        href="/main"
      >
        <button
          class="sc-ksdxAp ccjrxe"
        >
          Try Taskter today
        </button>
      </a>
    </div>
    <div
      class="sc-bkkfTU bycOoO"
    >
      <img
        class="sc-fotPbf jxoOJh"
        src="LandingImage.png"
      />
    </div>
  </div>
  <div
    class="sc-bdvvaa fPqNUM"
  >
    <div
      class="sc-gsDJrp hILqJY"
    >
      <div
        class="sc-dkPtyc havONB"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root"
          focusable="false"
          style="font-size: 80px; color: rgb(115, 194, 251);"
          viewBox="0 0 24 24"
        >
          <path
            d="M22 5.72l-4.6-3.86-1.29 1.53 4.6 3.86L22 5.72zM7.88 3.39L6.6 1.86 2 5.71l1.29 1.53 4.59-3.85zM12.5 8H11v6l4.75 2.85.75-1.23-4-2.37V8zM12 4c-4.97 0-9 4.03-9 9s4.02 9 9 9c4.97 0 9-4.03 9-9s-4.03-9-9-9zm0 16c-3.87 0-7-3.13-7-7s3.13-7 7-7 7 3.13 7 7-3.13 7-7 7z"
          />
        </svg>
      </div>
      <div
        class="sc-eCImvq giJlSO"
      >
        On the go? We can help!
      </div>
      <div
        class="sc-hKwCoD iPwudq"
      >
        <div
          class="sc-gKckTs kRpptL"
        >
          <img
            class="sc-jRQAMF eMhOwQ"
            src="taskPic.png"
          />
          <div
            class="sc-iCfLBT iplDYQ"
          >
            Organize your work flow
          </div>
        </div>
        <div
          class="sc-gKckTs kRpptL"
        >
          <img
            class="sc-furvIG wdos"
            src="mobilePic.png"
          />
          <div
            class="sc-iCfLBT iplDYQ"
          >
            Use our app
          </div>
        </div>
      </div>
    </div>
  </div>
  <footer
    class="sc-pVTma gCNFvr"
  >
    <div>
      <h1
        class="sc-jrQzUz ftPwLW"
      >
        Give us a try
      </h1>
      <a
        href="/main"
      >
        <button
          class="sc-kDThTU itaFPV"
        >
          Try Taskter today
        </button>
      </a>
    </div>
    <img
      class="sc-iqsfdx dkcVEi"
      src="taksListPic.png"
    />
  </footer>
</DocumentFragment>
`;
